### PR TITLE
call helper file "setup"

### LINKED ISSUE #183: Should use_vcr() create a file called setup not helper
https://testthat.r-lib.org/reference/test_dir.html#special-files

### PR DESCRIPTION
**Pull request by maelle**
Fix #183

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -10,13 +10,14 @@ vcr 0.5.4
 
 * changed `write_disk_path` handling internally to not run it through `normalizePath` before recording it to the cassette; passing the path through `normalizePath` was leading to the full path recorded in the cassette, which means in a package testing context that a test that uses a file on disk will (likely) only work on the machine the cassette was first created on. with relative paths in a package context, a test that has a file written on disk should now work in different testing contexts (locally, and various continuous integration platforms) (#135) (#166)
 * added a bit of documentation about large files created when using vcr, and how to ignore them if needed within `.Rinstignore` and/or `.Rbuildignore` (#164)
+* changed the name `helper-pkgname` to `setup-pkgname` to comply with `testthat`'s latest recommendations (#183, @maelle).
```
Comment (sckott) [inline NEWS.md:13]: this is the current version on cran - i create news items right before i submit to cran

**Review by sckott — CHANGES_REQUESTED** [review]
thanks, just a comment on the news file change

**Comment by maelle** [comment]
Wondering whether this was the best change https://github.com/r-lib/testthat/issues/1206#issuecomment-713120193

**Comment by maelle** [comment]
Probably ok https://testthat.r-lib.org/articles/test-fixtures.html#package

File: NEWS.md (hunk 1 above)
Comment (maelle) [reply NEWS.md]: oops, ok, I removed the item.
do you keep track of changes anywhere or do you just use the commit history?

Comment (sckott) [reply NEWS.md]: use the issues and use `git difftool` to look at changes since last git tag (which is the last CRAN version)

**Review by sckott — APPROVED** [review]

**Pull request merged**